docs/modules/ROOT/pages/object-storage/troubleshooting.adoc

```diff
@@ -0,0 +1,5 @@
+= Common Troubleshooting
```

Comment:
I suggest we separate this page into another PR, until the workflow is clearly defined.

Reply:
I've removed the file from this PR.